samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -815,14 +834,16 @@ static int configure_streams(void)
 
 static int create_group(void)
 {
-	struct bt_audio_stream *streams_p[ARRAY_SIZE(streams)];
+	struct bt_audio_unicast_group_param params[ARRAY_SIZE(streams)];
 	int err;
 
 	for (size_t i = 0U; i < configured_stream_count; i++) {
-		streams_p[i] = &streams[i];
+		params[i].stream = &streams[i];
```

Comment:
This is nice! so now either the developer can create unicast_group with predefined settings or can fetch the settings during configuration.